test/end-to-end/source-fetching-toggle.test.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
```

Comment:
Indentation is not correct in several places, e.g.,  `var reqParams =...` File ends with two closing braces on the same column.